metrics/web/lib/project_groups/presentation/widgets/project_checkbox_list.dart

```diff
@@ -11,11 +11,16 @@ import 'package:provider/provider.dart';
 class ProjectCheckboxList extends StatelessWidget {
   @override
   Widget build(BuildContext buildContext) {
+    const placeholderPadding = EdgeInsets.only(top: 16.0);
```

Comment:
Should it be 24 pixels padding here according to designs?

Reply:
@vova-beloded! This is because the widget above has the 8px bottom padding.